src/common/common.types.ts

```diff
@@ -23,15 +23,57 @@ interface Metadata {
 }
 
 interface Data {
```

Comment:
Added TSDoc for all fields in this type since it's being used on Admin now. Added `@deprecated` to `serie` and `origin`.